43/README.md

```diff
@@ -29,15 +30,12 @@ The ZRE protocol provides a way for a set of nodes on a local network to discove
 * To be neutral with respect to operating system, programming language, and hardware.
 * To allow any number of nodes to run in one process, to allow large-scale simulation and testing.
 
-## Changes Over Version 1
+## Changes Over Version 2
 
-This is version 2 of ZRE. The changes over version 1 are:
+This is version 3 of ZRE. The changes over version 2 are:
 
-* The 'strings' and 'dictionary' types redesigned for better expandability.
-* The version number was increased to 2.
-* The hard-coded dependency on TCP transport has been removed (merging ipaddress and port into a single endpoint field).
-* Removed the logging extension, which is better handled by a separate framework.
-* Protocol signature is 1 rather than 0.
+* The version number was increased to 3.
+* Becon packets support advertising of a 32-octet CURVE public key for the purposes of establishing CURVE enabled security to the connection.
```

Comment:
thanks for the comments! i think that's a good idea and will update. the short and skinny:

+ i decided to leave authorization out of the lower level (it's complicated, and there's ZAP and and and.. IRL we're looking at implementing this higher up the stack, just not quite sure yet)

+ all peers are ... not? required to have a unique keypair (good thing to note probably) but probably should add that as a SHOULD (?), want to make it easy to get going, but with a pathway towards "good security" using small steps (eg: step 1, get it working, step 2, find a keypair distribution pattern that works, step 3, think about authorization, ...)

+ benefits- will add some of the obvious stuff about general TLS, but with a bent towards "beacon is usually on a local lan, why do i care?" which is a realistic question. the gist of it was enabling people to test ZYRE on a lan with TLS since CURVE can be hard to learn. we're also looking at this from some of our ipv6 networks and maybe branching out to using a multi-cast style beacon framework too. figured encrypting / testing with simple udp beacon was an easy way to figure out some of the nuances and branch out from there.

all good questions, will try to incorporate some responses to help address those.